runtime/themes/gruvbox_light.toml

```diff
@@ -20,6 +20,7 @@
 "function" = { fg = "green1", modifiers = ["bold"] }
 "function.macro" = "aqua1"
 "function.builtin" = "yellow1"
+"tag" = "red1"
```

Comment:
Why is tag in gruvbox light but not in gruvbox?

Reply:
Ahh, missed this. Will send a patch. Thanks for pointing it out 🙇🏽

Reply:
Done @pickfire: https://github.com/helix-editor/helix/pull/1555